<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWidget</class>
 <widget class="QWidget" name="StartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="plannerButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Планировщик</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="kakeboButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Финансы</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="collectionButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Коллекции</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
